Libs/MRML/Core/vtkMRMLClipModelsNode.cxx

```diff
@@ -94,6 +96,12 @@ void vtkMRMLClipModelsNode::ReadXMLAttributes(const char** atts)
       ss << attValue;
       ss >> ClipType;
       }
+    else if (!strcmp(attName, "clippingMethod"))
+      {
+      std::stringstream ss;
+      ss << attValue;
+      ss >> ClippingMethod;
```

Comment:
Yes, it's very important to not use magic numbers in the scene file (but write associated string instead). Magic numbers makes the scene file not-human-readable and make future updates difficult (hard to maintain backward compatibility).